resources/opentelemetry-kw-telemetry-values.yaml

```diff
@@ -0,0 +1,9 @@
+telemetry:
+  enabled: True
+  metrics:
+    port: 8080
+  tracing:
+    jaeger:
+      endpoint: "my-open-telemetry-collector.jaeger.svc.cluster.local:14250"
+      tls:
```

Comment:
doh, just realized, `tls.insecure` is unreleased. It will be in charts appVersion 1.3.0 (the `rc`s that are already released as charts).

Maybe that caused the problems on failing to get metrics and traces.